lib/devise-authy/models/verify_mfaable.rb

```diff
@@ -0,0 +1,19 @@
+module Devise
+
+  module Models
+
+    # Handles connecting the MfaConfig to the resource
+    module VerifyMfaable
```

Comment:
This could alternatively be named `MfaVerifiable`... if you think that's clearer, I'd say lets go for a rename. Otherwise, if there's an advantage to keeping `Verify______` naming as a pattern, let's keep it :)